third_party/terraform/tests/resource_compute_instance_template_test.go.erb

```diff
@@ -787,6 +787,35 @@ func TestAccComputeInstanceTemplate_shieldedVmConfig2(t *testing.T) {
 	})
 }
 
+<% unless version == 'ga' -%>
+func TestAccComputeInstanceTemplate_ConfidentialInstanceConfigMain(t *testing.T) {
+	t.Parallel()
+
+	var instanceTemplate computeBeta.InstanceTemplate
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
```

Comment:
If you use `testAccProvidersOiCS` here instead of `testAccProviders`, you should have access to the `google-beta` provider and that should circumvent the error you're seeing. This is applicable for both tests.

Reply:
thanks, this made the tests run in the beta provider.